x/uibc/client/tests/query.go

```diff
@@ -0,0 +1,73 @@
+//go:build experimental
```

Comment:
Why this file doesn't have `_test.go` suffix?

Reply:
No need `_test` suffix, `cli_test` will now call the `quert` file test funs